tests-fuzz/targets/fuzz_insert_logical_table.rs

```diff
@@ -160,7 +161,39 @@ async fn execute_insert(ctx: FuzzContext, input: FuzzInput) -> Result<()> {
         }
     );
 
-    // TODO: Validate inserted rows
+    // Validate inserted rows
+    let ts_column_idx = create_logical_table_expr
+        .columns
+        .iter()
+        .position(|c| c.is_time_index())
+        .unwrap();
+    let ts_column_name = create_logical_table_expr.columns[ts_column_idx]
+        .name
+        .clone();
+    let ts_column_idx_in_insert = insert_expr
+        .columns
+        .iter()
+        .position(|c| c.name == ts_column_name)
+        .unwrap();
+    let column_list = insert_expr
+        .columns
+        .iter()
+        .map(|c| c.name.to_string())
+        .collect::<Vec<_>>()
+        .join(", ")
+        .to_string();
+    let select_sql = format!(
+        "SELECT {} FROM {} ORDER BY {}",
+        column_list, create_logical_table_expr.table_name, ts_column_name
+    );
+    let fetched_rows = validator::row::fetch_values(&ctx.greptime, select_sql.as_str()).await?;
```

Comment:
I tried to construct the sql inside the `fetch_values` function, but failed in some lifetime issues. I wonder if it's possible to construct the sql inside.